source/src_io/write_input.cpp

```diff
@@ -226,55 +206,17 @@ void Input::Print(const std::string &fn)const
 			<< vdw_period.z<< " #periods of periodic structure" << std::endl;
 	
 	
-	ofs << "\n#Parameters (15.spectrum)" << std::endl;              // pengfei Li add 2016-11-23
-	ModuleBase::GlobalFunc::OUTP(ofs,"spectral_type",spectral_type,"the type of the calculated spectrum");
-	ModuleBase::GlobalFunc::OUTP(ofs,"spectral_method",spectral_method,"0: tddft(linear response)");
-	ModuleBase::GlobalFunc::OUTP(ofs,"kernel_type",kernel_type,"the kernel type: rpa, tdlda ...");
-	ModuleBase::GlobalFunc::OUTP(ofs,"eels_method",eels_method,"0: hilbert_transform method; 1: standard method");
-	ModuleBase::GlobalFunc::OUTP(ofs,"absorption_method",absorption_method,"0: vasp's method  1: pwscf's method");
-	ModuleBase::GlobalFunc::OUTP(ofs,"system",system_type,"the calculate system");
-	ModuleBase::GlobalFunc::OUTP(ofs,"eta",eta,"eta(Ry)");
-	ModuleBase::GlobalFunc::OUTP(ofs,"domega",domega,"domega(Ry)");
-	ModuleBase::GlobalFunc::OUTP(ofs,"nomega",nomega,"nomega");
-	ModuleBase::GlobalFunc::OUTP(ofs,"ecut_chi",ecut_chi,"the dimension of chi matrix");
-	ofs << std::setw(20) <<"q_start"<<q_start[0]<<"   "
-		<<q_start[1]<<"   "<<q_start[2]
-		<<"  #the position of the first q point in direct coordinate" <<std::endl;
-	ofs << std::setw(20) <<"q_direction"<<q_direct[0]<<"   "<<q_direct[1]<<"   "<<q_direct[2]<<"  #the q direction" <<std::endl;
-	ModuleBase::GlobalFunc::OUTP(ofs,"nq",nq,"the total number of qpoints for calculation");
-	ModuleBase::GlobalFunc::OUTP(ofs,"out_epsilon",out_epsilon,"output epsilon or not");
-	ModuleBase::GlobalFunc::OUTP(ofs,"out_chi",out_chi,"output chi or not");
-	ModuleBase::GlobalFunc::OUTP(ofs,"out_chi0",out_chi0,"output chi0 or not");
-	ModuleBase::GlobalFunc::OUTP(ofs,"fermi_level",fermi_level,"the change of the fermi_level(Ry)");
-	ModuleBase::GlobalFunc::OUTP(ofs,"coulomb_cutoff",coulomb_cutoff," turn on the coulomb_cutoff or not");
-	ModuleBase::GlobalFunc::OUTP(ofs,"kmesh_interpolation",kmesh_interpolation,"calculting <i,0|j,R>");
-	for(int i=0; i<nq; i++)
-	{
-		ofs << std::setw(20) <<"qcar" << qcar[i][0] <<"   "<< qcar[i][1] <<"   "<<qcar[i][2]<<"  #(unit: 2PI/lat0)" << std::endl;
-	}
+	ofs << "\n#Parameters (14.spectrum)" << std::endl;              // pengfei Li add 2016-11-23
```

Comment:
Parameters of spectrum are deleted, but the title for spectrum parameters is still alive?

Reply:
As can be seen in the next conversation, there are some parameters remained in the section, so I retained the section `spectrum`.